subsys/bluetooth/services/Kconfig.hids

```diff
@@ -48,6 +48,25 @@ config BT_GATT_HIDS_FEATURE_REP_MAX
 	help
 	  Maximum number of HIDS Feature Reports that can be set for HIDS.
 
+choice
+	prompt "Defalut permissions used for HID attributes"
+	default BT_GATT_HIDS_DEFAULT_PERM_RW
```

Comment:
HID over GATT requires at least Encryption.
Should we set that as default here? or should we have applications/nrf_desktop (maybe the samples peripheral_hids_* as well) set the Kconfig option?

Reply:
I have a story to rise the level for our app. I would prefer to set it there so we are not dependent on the default setting that may change for some reason in the future.

Reply:
Sounds good.